<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task_edit</class>
 <widget class="QWidget" name="task_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>任务编辑</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_type">
       <property name="text">
        <string>文件复制方向：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="RadioButton" name="radio_type_single">
       <property name="text">
        <string>单向</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btngp_sync_type</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="RadioButton" name="radio_type_double">
       <property name="text">
        <string>双向</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">btngp_sync_type</string>
       </attribute>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="grpbox_src">
       <property name="title">
        <string>文件源</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="ListWidget" name="listwidget_src"/>
        </item>
        <item row="1" column="0">
         <widget class="PushButton" name="btn_src_del">
          <property name="text">
           <string>删除</string>
          </property>
          <property name="lightCustomQss" stdset="0">
           <string>PushButton{background-color:#e81123;}
PushButton:hover{background-color:#e63342;}</string>
          </property>
          <property name="darkCustomQss" stdset="0">
           <string>PushButton{background-color:#e81123;}
PushButton:hover{background-color:#e63342;}</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="PrimaryPushButton" name="btn_src_add">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="grpbox_dst">
       <property name="title">
        <string>目标路径</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="PushButton" name="btn_select_dst_path">
          <property name="text">
           <string>选择路径</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="LineEdit" name="lineedit_dst_path"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="grpbox_task">
       <property name="title">
        <string>任务属性</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_interval">
            <property name="text">
             <string>执行间隔：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TimePicker" name="timeedit_interval"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_repeat">
            <property name="text">
             <string>重复文件：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="RadioButton" name="radio_repeat_replace">
            <property name="text">
             <string>替换</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">btngp_repeat_file</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="RadioButton" name="radio_repeat_ask">
            <property name="text">
             <string>询问</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">btngp_repeat_file</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="RadioButton" name="radio_repeat_ignore">
            <property name="text">
             <string>跳过</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">btngp_repeat_file</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_show_result">
            <property name="text">
             <string>执行后弹出运行结果：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="RadioButton" name="radio_show_result_error">
            <property name="text">
             <string>仅错误</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">btngp_show_result</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="RadioButton" name="radio_show_result_always">
            <property name="text">
             <string>始终</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">btngp_show_result</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="grpbox_file">
       <property name="title">
        <string>文件/文件夹属性</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_file_count">
          <property name="text">
           <string>文件数：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_file_count_value">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>未知</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_size_count">
          <property name="text">
           <string>文件总大小：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_size_count_value">
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>未知</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="SwitchButton" name="ckb_include_later">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>包含之后增加的文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="PushButton" name="btn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PrimaryPushButton" name="btn_save">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RadioButton</class>
   <extends>QRadioButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ListWidget</class>
   <extends>QListWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TimePicker</class>
   <extends>QTimeEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QCheckBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btngp_sync_type"/>
  <buttongroup name="btngp_repeat_file"/>
  <buttongroup name="btngp_show_result"/>
 </buttongroups>
</ui>
